gateleen-queue/src/test/java/org/swisspush/gateleen/queue/mocks/FastFailMultiMap.java

```diff
@@ -0,0 +1,151 @@
+package org.swisspush.gateleen.queue.mocks;
```

Comment:
Move to the _gateleen-core_ package like the _DummyHttpServerRequest_ and _DummyHttpServerResponse_ classes because this class could be useful elsewhere too.